cli/dcoscli/main.py

```diff
@@ -79,6 +85,10 @@ def _main():
        not auth.check_if_user_authenticated():
         auth.force_auth()
 
+    #  TODO (tamar): support reading tuple:
+    #  (/path/to/cert, /path/to/private/key)
```

Comment:
Are we going to implement this before merging this PR? If not then let's remove the TODO and create an Issue.